packages/patternfly-4/react-core/src/components/Select/SelectOption.tsx

```diff
@@ -18,7 +19,7 @@ export interface SelectOptionProps extends Omit<React.HTMLProps<HTMLElement>, 't
   index?: number;
   /** Indicates which component will be used as select item */
   component?: React.ReactNode;
-  /** The value for the option, if passing an object you most provide a toString function */
+  /** The value for the option, if passing an object you most provide a toString function and optional compareTo function */
```

Comment:
If we add the comments to the interface we could just tell them see the SelectOptionObject.  E.g. The value for the option, can be a string or a SelectOptionObject.